packages/select/src/Select/components/index.ts

```diff
@@ -1 +1,3 @@
-export type { SingleTriggerButtonProps } from "./Trigger"
+export type { TriggerButtonProps } from "./TriggerButton"
```

Comment:
Is there a reason why we're only exporting types and not components here?

Reply:
From the Select>index.ts file 

note: This was taken from the FilterMultiSelect file structure

// Here only export component props as we would like to enforce consumer to use dot notation component.
export * from "./components"

Reply:
Haha. There's a slight difference for this. Exporting to the index within `components` does not necessarily mean it's exported in the package `index`. The ones in the `components` folder are meant for internal use, and the ones in the `src` folder are the actual entrypoint that we recommend consumers import from (where you can do the restricting).
Not a big deal though. Whatever works 🤷🏻‍♀️